src/components/shared/Validations.tsx

```diff
@@ -14,8 +16,8 @@ export const Validations: React.FunctionComponent<IValidations> = ({
   required,
   validations: { deprecated, readOnly, writeOnly, ...validations },
 }) => {
+  const validationCount = Object.keys(validations).filter(validation => validation !== 'format').length;
```

Comment:
Alternatively, let's do
```suggestion
const filteredValidations = validations.filter(validation => validation !== 'format');
const validationCount = Object.keys(filteredValidations).length; 
```

Reply:
I ended separating `format` from `validations` like this:

```
validations: { deprecated, readOnly, writeOnly, format, ...validations }
```